terraform/version_constraint.tf

```diff
@@ -0,0 +1,3 @@
+terraform {
+  required_version = ">= 0.8.5"
```

Comment:
This is currently replicating the existing constraint that was in place. I think if we want to alter the versions we constrain this to, that's a separate change, and should be a separate PR.